### PR TITLE
Fixing small bugs

### LINKED ISSUE #282: Allow mirroring of "Body" preview (or, let me have Right be on the Right)
Even after using the new UI for weeks, I still get my left and right confused with this layout. 

image

### LINKED ISSUE #612: Tracker warning with waist/hip only
Using a waist or hip tracker only (no legs), the user should not get a warning about a missing chest tracker.

### LINKED ISSUE #917: Onboarding: Add WiFi Password length minimum/maximum
The minimum and maximum length for the WiFi password is not checked.
A WiFi Password can be from minimum 8bytes to maximum 63bytes long.

If it is shorter than 8bytes it will fail to connect.

### PR DESCRIPTION
**Pull request by ImUrX**
* Fixes skeleton preview modifying tracker preview variable
* Fix where server linux logs are saved
* Add tip for turning on tracker when doing onboarding
* Fix #917 and add support for no password
* Add mirror view toggle for body assignment as some people mix it up (Fix #282)
* Add autobone video guide
* Fix #612 
* Fix server not running with nix file (libusb missing)
* Add warning that you need to only measure your body once

### CODE REVIEW

**Review by ButterscotchV — APPROVED** [review]

**Review by Louka3000 — APPROVED** [review]

File: gui/public/i18n/en/translation.ftl [hunk 1]
```diff
@@ -19,6 +19,7 @@ tips-find_tracker = Not sure which tracker is which? Shake a tracker and it will
 tips-do_not_move_heels = Ensure your heels do not move during recording!
 tips-file_select = Drag & drop files to use, or <u>browse</u>.
 tips-tap_setup = You can slowly tap 2 times your tracker to choose it instead of selecting it from the menu.
+tips-turn_on_tracker = Remember to <b><em>turn on your tracker</em></b> before connecting it to the PC
```
Comment (ButterscotchV) [inline gui/public/i18n/en/translation.ftl:22]: Shouldn't we suggest turning it on after? I've seen people have mysterious issues by turning it on before.
```suggestion
tips-turn_on_tracker = Remember to <b><em>turn on your tracker</em></b> after connecting it to the PC
```

Comment (ButterscotchV) [reply gui/public/i18n/en/translation.ftl:22]: Nah it's dangerous to have DIY trackers on while plugged in at all, no difference for when it's turned on

Comment (ImUrX) [reply gui/public/i18n/en/translation.ftl:22]: I changed the string, I think this can be resolved now?

**Review by ButterscotchV — APPROVED** [review]

**Review by Louka3000 — APPROVED** [review]

File: gui/src/components/onboarding/pages/WifiCreds.tsx [hunk 2]
```diff
@@ -77,7 +77,14 @@ export function WifiCredsPage() {
             >
               <Input
                 control={control}
-                rules={{ required: true }}
+                rules={{
+                  validate: {
+                    validPassword: (v: string | undefined) =>
+                      v === undefined ||
+                      v.length === 0 ||
+                      new Blob([v]).size >= 8,
```
[the remark this answers is not in the document]
Comment (ImUrX) [reply gui/src/components/onboarding/pages/WifiCreds.tsx:85]: yes [`String.length`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length) is based on utf-16 code units. `"😀".length` is 2 when in bytes it's 4

[the remark this answers is not in the document]
Comment (ButterscotchV) [reply gui/src/components/onboarding/pages/WifiCreds.tsx:85]: That's what I'd thought, sounds good then

**Review by ButterscotchV — APPROVED** [review]

**Pull request merged**